run-tests.sh

```diff
@@ -1,32 +1,19 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
 # -*- coding: utf-8 -*-
 #
 # This file is part of Invenio.
-# Copyright (C) 2016 CERN.
+# Copyright (C) 2015-2020 CERN.
 #
-# Invenio is free software; you can redistribute it
```

Comment:
We are technically changing the license of the package here... I have added an issue to do a complete migration to GPL to MIT in https://github.com/inveniosoftware/invenio-csl-rest/issues/19.